lib/qb_integration/processors/invoice.rb

```diff
@@ -53,7 +53,10 @@ def as_flowlink_hash
           linked_transactions: build_linked_transactions(invoice.linked_transactions),
           custom_fields: build_custom_fields(invoice.custom_fields),
           line_items: build_line_items(invoice.line_items),
-          status: determine_status
+          status: determine_status,
+          relationships: [
+            { object: "customer", key: "id" }
```

Comment:
I think it is better to use system specific ids, like qbo_id, and if not present add it in the payload.